tools/CustomTasks/DependencyPropertyCodeGen.cs

```diff
@@ -468,6 +472,11 @@ private string GetPropertyTypeOverride(params MemberInfo[] members)
             return GetAttributeValue<string>("MUXPropertyTypeAttribute", members);
         }
 
+        private string GetPropertyIgnoredByCodeGen(params MemberInfo[] members)
+        {
```

Comment:
Regenerating this needs an updated nuget package pushed to the feed and the projects updated to use them. But I don't see this new property being used ?

Reply:
Looks like i forgot to remove all of the changes here, its from when I was experimenting with having an attached and instance state property

---
In reply to: [580451020](https://github.com/microsoft/microsoft-ui-xaml/pull/4277#discussion_r580451020) [](ancestors = 580451020)